app/scripts/directives/createSecret.js

```diff
@@ -70,6 +70,15 @@ angular.module("openshiftConsole")
                 label: "Webhook Secret"
               }
             ]
+          },
+          generic: {
+            label: "Generic Secret",
+            authTypes: [
+              {
+                id: "Generic",
```

Comment:
```
id: "Opaque",
```

Even if we call it generic secret in the UI, the ID should be Opaque

Reply:
ok, fixed